pkg/lib/oauth/handler/handler_authz.go

```diff
@@ -586,6 +586,9 @@ func (h *AuthorizationHandler) handleSettingsAction(
 ) (httputil.Result, error) {
 	redirectURI, err := h.UIURLBuilder.BuildSettingsActionURL(client, r, oauthSessionEntry)
 	if err != nil {
+		if apierrors.IsKind(err, oidc.ErrInvalidSettingsAction) {
```

Comment:
I suggest we change ErrInvalidSettingsAction to be a protocol error, so that we will not forget the translation (as there is no translation anymore).